app/js/templates.js

```diff
@@ -181,12 +181,14 @@ frame.set("loop.first", t_1 === 0);
 frame.set("loop.last", t_1 === t_2 - 1);
 frame.set("loop.length", t_2);
 output += "\n                <option ";
-if(t_4 == runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "user")),"locale", env.autoesc)) {
+if(runtime.memberLookup((t_4),0, env.autoesc) == runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "user")),"locale", env.autoesc)) {
```

Comment:
I think we should delete this file. But that involves working with QA, so it will have to be done a different time.